### PR TITLE
Added Dairy website

### PR DESCRIPTION
**Pull request by SamarthKadam**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes:  #801 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
# Checklist:

- [x] I have made this from my own
- [x] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK

### CODE REVIEW

File: DairyProductWebsite/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (VidhiBhatt01) [inline DairyProductWebsite/index.html:7]: Add suitable title

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Make it responsive and add the project in style_links.json.
image
image

**Comment by SamarthKadam** [comment]
@VidhiBhatt01  I've made the required changes

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Everything is perfect but this looks weird.

image

**Comment by SamarthKadam** [comment]
@VidhiBhatt01 It looks perfect from my side. Can we please compromise this? I got a lot of issues pending as this is last week of coding period so Please

**Review by VidhiBhatt01 — APPROVED** [review]
Cool

**Pull request merged**